docs/fr/manuals/addressing.md

```diff
@@ -0,0 +1,242 @@
+label.set_text("my_gameobject#my_label", "Hello World!")
+
+local pos = go.get_position("my_gameobject")
+go.set_position(pos, "/level/stuff/other_gameobject")
+
+msg.post("#", "hello_there")
+local id = go.get_id(".")
+```
+
+Commençons par un exemple très simple. Vous avez un GameObject avec un seul sprite (un composant du nom de "body"). Vous disposez également d'un script (un autre composant dont le nom est "controller") pour contrôler le GameObject. La configuration dans l'éditeur ressemble à ceci:
+
+![image](https://github.com/unlitcolor/doc/assets/9135915/287609c7-32e4-43cc-bc79-3ea9ca065805)
+
+Pour désactiver le sprite au démarrage du jeu et le faire apparaître plus tard, cela se fait facilement en plaçant le code suivant dans "controller.script":
+
+```lua
+function init(self)
+    msg.post("#body", "disable") -- <1>
+end
+```
+1. Ne vous préoccupez pas du caractère «#», nous y reviendrons plus tard.
+
+Tout fonctionne comme prévu. Lorsque le jeu démarre, le composant script *adresse* le composant sprite via son identifiant "body" et utilise cette adresse pour lui envoyer un *message* contenant "disable" (désactiver). L'effet de ce message spécial est que le sprite cache son graphisme. Schématiquement, cela ressemble à ça:
+
+![image](https://github.com/unlitcolor/doc/assets/9135915/c9851e7b-8365-4854-a49c-a225bfcbfdb9)
+
+Les identifiants dans la configuration sont arbitraires. Ici, nous avons choisi de donner au GameObject l'id «bean» (haricot), son sprite a été nommé «body», et le script qui contrôle le personnage s'appelle «contrôleur».
+
+::: sidenote
+<br>Si vous ne mettez pas un nom à l'id, l'éditeur le fera. Chaque fois que vous créez un nouveau GameObject ou composant, une propriété *Id* unique est automatiquement définie.
+
+- Les GameObjects reçoivent automatiquement un identifiant appelé "go" avec un énumérateur ("go2", "go3" etc).
+- Les composants reçoivent un identifiant correspondant à leur type ("sprite", "sprite2" etc).
+
+Vous pouvez vous en tenir à ces noms attribués de manière automatique si vous le souhaitez, mais nous vous encourageons à les remplacer par des noms plus appropriés et descriptifs.
+:::
+
+Ajoutons un nouveau sprite et donnons un bouclier au haricot (bean):
+
+![image](https://github.com/unlitcolor/doc/assets/9135915/79f3e091-0414-46eb-80d9-485da7e64842)
+
+Ce nouveau composant doit être identifié de manière unique dans le GameObject. Si vous lui donnez le nom «body» (corps), le script ne comprendra pas quel sprite devra recevoir le message «disable». C'est pourquoi nous choisissons un id unique (et descriptif) "shield" (bouclier). Nous pouvons désormais activer et désactiver les sprites «body» et «shield» à volonté.
+
+![image](https://github.com/unlitcolor/doc/assets/9135915/6a0c44dd-6aa1-45b3-a30c-14ec9ea1efd5)
+
+::: sidenote
+<br>Si vous utilisez un même identifiant plus d'une fois, l'éditeur signalera une erreur, donc vous n'aurez jamais de problème: :::
```

Comment:
I think the ::: needs to be on a separate line. Refer to the English version.

Reply:
> I think the ::: needs to be on a separate line. Refer to the English version.

In the English version, :::: is not on a separate line, but there's no separate line <br> between the text and :::: sidenote, I'll just remove the <br> so it will look exactly the same.